example.init.yaml

```diff
@@ -115,17 +135,17 @@ slack:
   url: http://gateway.openfaas:8080/function/echo
 
 ### Users allowed to access your OpenFaaS Cloud
-#### ACL for your users, it must be a raw text file or GitHub RAW URL	
+#### ACL for your users, it must be a raw text file or GitHub RAW URL
 customers_url: "https://raw.githubusercontent.com/openfaas/openfaas-cloud/master/CUSTOMERS"
 
 ##  S3 configuration
 ### Build logs from buildkit are stored in S3
 ### Defaults to in-cluster deployment of Minio. AWS S3 is also possible
-s3:
-  s3_url: cloud-minio.openfaas.svc.cluster.local:9000
-  s3_region: us-east-1
-  s3_tls: false
-  s3_bucket: pipeline
+#s3:
```

Comment:
I think that `s3` should always be on and populated with the defaults. It's also a non-optional feature, so we don't need to have a filter for it. (IMHO) What are your thoughts?

Reply:
Yes, you are right I wanted to get your review on those parts to be sure. 👍

Reply:
Will add `default` filter for those cases

Reply:
Done 👍